### PR TITLE
Add update to organizers profile

### PR DESCRIPTION
**Pull request by dorianpula**
- Add details to the organizer page.

Outstanding:
- Need to add photo

### CODE REVIEW

File: data/organizers.json [hunk 1]
```diff
@@ -14,6 +14,15 @@
   "instagram": "myles",
   "github": "myles",
   "photo": "myles-braithwaite.png"
+}, {
+  "first_name": "Dorian",
+  "last_name": "Puła",
+  "title": "Software Development Engineer",
+  "company": "Points",
+  "bitbucket": "https://bitbucket.org/dorianpula/",
```
Comment (myles) [inline data/organizers.json]: Could you change this to be just your username.

Comment (dorianpula) [reply data/organizers.json]: Sure!  I was just experimenting with how the links were rendered in the preview.

**Comment by dorianpula** [comment]
@myles Do we get a maximum of two links under our profiles?  (I'd rather just go with my BitBucket links compared to my Github one, if that is the case.)

**Comment by dorianpula** [comment]
Thanks @myles!

**Pull request merged**